packages/frontend/app/routes/assets.$assetId.tsx

```diff
@@ -170,12 +175,16 @@ export default function ViewAssetPage() {
                     type='number'
                     name='basisPoints'
                     label='Basis Points'
-                    defaultValue={asset.sendingFee?.basisPoints ?? undefined}
                     error={response?.errors.sendingFee.fieldErrors.basisPoints}
+                    value={basicPointsInput}
+                    onChange={(e) =>
+                      setBasicPointsInput(parseFloat(e?.target?.value))
+                    }
                   />
                   <p className='text-gray-500 text-sm mt-2'>
                     A single basis point is a fee equal to 0.01% of the total
-                    amount. A fee of 1 basis point on $100 is $0.01.
+                    amount. A fee of {basicPointsInput || 1} basis point on $100
+                    is ${(basicPointsInput || 1) * 0.01}.
```

Comment:
```suggestion
                    is ${(basisPointsInput || 1) * 0.0001}.
```

Reply:
0.01% = 0.0001

Reply:
It would be better to use a set fixed-point notation to display the value as well since working with floating-point numbers in JavaScript can have precision issues. E.g. if you try a basis point fee of 144 the text says, "A fee of 144 basis point on $100 is $0.014400000000000001."
We can fix this issue with toFixed(4)
```suggestion
                    is ${((basisPointsInput || 1) * 0.0001).toFixed(4)}.
```

Reply:
I think the calculation is correct, since it's `0.0001 * 100($) * basisPoints = 0.01 * basisPoints`. But the `toFixed` should be added 👍